openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -27,6 +27,7 @@ compiler/tiered/Level2RecompilationTest.java https://github.com/adoptium/aqa-tes
 # compiler/6894807/IsInstanceTest.java https://github.com/adoptium/adoptium/issues/58 aix-all
 compiler/6894807/IsInstanceTest.java https://github.com/adoptium/aqa-tests/issues/2818 aix-all
 compiler/8004051/Test8004051.java https://github.com/adoptium/aqa-tests/issues/2818 aix-all
+compiler/8009761/Test8009761.java https://github.com/adoptium/aqa-tests/issues/5221 linux-ppc64le
```

Comment:
> Is there an upstream **open** issue like https://bugs.openjdk.org/browse/JDK-8021775 pointing to the issue on jdk8?

I searched, and didn't find an open upstream issue. There was one for the backport 10 years back, but it got deferred to JDK8u20, and (it appears) completely forgotten.

Lots of Test8009761 issues, but they're all closed or resolved.

> If so, then please use that issue instead of 5221 as the referring issue. If not we ultimately should raise it and refer to it (and ultimately this test excluded upstream). 

Agreed. I left a note for myself [here](https://github.com/adoptium/aqa-tests/issues/5221#issuecomment-2059019895), to be actioned after the release.

> In the meantime, we can bring this change in.

👍